home.html

```diff
@@ -0,0 +1,132 @@
+         <li><i class="fa fa-instagram"></i></li> 
+         <li><a href="">French</a></li>
+         <li><a href="">My page</a></li>
+         <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <div class="nav-bar">
+        <a href="home.html"><img class="click" src="images/logo.png"></a>
+        <div class="nav-links">
+          <a class="LINK" href="about.html">About</a>
+          <a class="LINK" href="#products">Our Products</a>
+          <a class="LINK"href="#Artists">Artists</a>
+          <a href="" class="flawless">FLAWLESS</a>
+        </div>
+      </div>
+      <i class="fa fa-bars fa-2x" id="bars"></i>
+    </nav>
+    <div class="nav-menu">
+      <i class="fa fa-times fa-2x" id="closingIcon"></i>
+      <ul>
+        <li class="mobileLinks"><a href="home.html">Home</a></li>
+        <li class="mobileLinks"><a href="#products">Products</a></li>
+        <li class="mobileLinks"><a href="about.html">About</a></li>
+      </ul>
+    </div>
+    <div class="main">
+      <q>Hi there!</q>
+      <h1>With Flawless, Glow like never before!</h1>
+      <p class="description">
+        Looking young and beautiful never comes without using the best cosmetics.
+        Here we have compiled the best skincare products by the best brands.
+      </p>
+      <p class="promotion">
+        10% OFF FOR FIRST PURCHASE
+      </p>
+      <a class="more" href="#viewmore">SEE OUR COLLECTIONS.</a>
+    </div>
+  </header>
+  <section class="products" id="products">
+    <h2>Main Products</h2>
+    <hr class="title-underline">
+    <div class="product-list">
+      <div class="skincare">
+        <img class="product-image" src="images/cleanser.png">
+        <h3>Cleansers</h3>
+        <p class="explication">Our cleansers help remove excess oil, makeup, sweat, dirt and dead skin.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/serum.png">
+        <h3>Serums</h3>
+        <p class="explication">By using our serums, your skin will be more radiant,full of colour and it'll be less dry.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/bodylotion.png">
+        <h3>Lotions</h3>
+        <p class="explication">Usse our body lotions daily and you'll notice a difference in your skin's hydration levels.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/nails.png">
+        <h3>Nails</h3>
+        <p class="explication">Use our nail products to ensure that your nails are in good condition and the skin on hands and feet is well cared.</p>
+      </div>
+      <div class="skincare">
+        <img class="product-image" src="images/hair.png"> 
+        <h3>Hair</h3>
+        <p class="explication">Our hair products will strengthen your hair follicles and enhance hair growth.</p>
+      </div>
+    </div>
+    <a class="button">View more products</a>
+  </section>
```

Comment:
Kindly implement the `hover` feature for the project section so you have a better UI

Design given :
![image](https://github.com/mariegrace31/capstone_one/assets/20537712/1b1e1eb0-3a0b-47d3-aab1-6619e04c9131)

your design :
![image](https://github.com/mariegrace31/capstone_one/assets/20537712/543f5273-21cb-4843-86b3-7c499d814462)